src/components/Card/CommunityCard.tsx

```diff
@@ -0,0 +1,37 @@
+import classNames from "classnames";
+import Link from "next/link";
+import { ReactNode } from "react";
+import { Card } from "./Card";
+interface CommunityCardProps {
+  className?: string;
+  title?: string;
+  icon?: ReactNode;
+  description?: string;
+  link: string;
+}
+export const CommunityCard: React.FC<CommunityCardProps> = ({
+  className,
+  title,
+  icon,
+  description,
+  link,
+}) => {
+  return (
+    <>
+      <Card className={classNames("p-32", className)}>
+        <p className="text-white text-24 self-center font-bold mb-32">
+          {title}
+        </p>
+        <div className="self-center mt-64 scale-400"> {icon}</div>
+        <p className="text-white text-14 self-center text-left mt-64">
+          {description}
+        </p>
+        <Link href={link}>
```

Comment:
Please introduce new `Button` component, that will be used everywhere where we have buttons. Color of the button can be controlled by `mode` property that can accept values 
`Primary` -> Blue
`Secondary` -> Outlined
`Attention` -> Green